docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/NavItem.php

```diff
@@ -226,7 +226,12 @@ public function getMegamenuContent() {
     $referencedEntities = $megamenuFieldEntityReference->referencedEntities();
     $hasMegamenu = count($referencedEntities) > 0;
     if ($hasMegamenu) {
-      $megamenuMarkupEncoded = $megamenuFieldEntityReference->entity->get('body')->value;
+      if ($megamenuFieldEntityReference->entity->hasField('body')) {
+        $megamenuMarkupEncoded = $megamenuFieldEntityReference->entity->get('body')->value;
+      }
+      elseif ($megamenuFieldEntityReference->entity->hasField('field_raw_html')) {
```

Comment:
Just use field_raw_html for this, and then constrain the Site Section to only allow raw html. That will be more straightforward -- we are not going to have content blocks.